pkgs/development/compilers/gerbil/glow-lang.nix

```diff
@@ -0,0 +1,42 @@
+{ lib, fetchFromGitLab, gerbil-support, gerbilPackages, gerbil, ... }:
+rec {
+  pname = "glow-lang";
+  version = "unstable-2021-02-26";
+  git-version = "0.1.0";
+  softwareName = "Glow";
+  gerbil-package = "mukn/glow";
+  version-path = "version";
```

Comment:
Some whitespace in here wouldn't hurt.

Reply:
Not sure what whitespace you want where.

Reply:
I would bet on vertical whitespace, like a newline right after that position.

Reply:
Yeah, but I need better guidelines than that, like some principle. I have 8 files like this one, and possibly more coming in the near future. I'm not going to have this one break the template. I can update all those files to follow an updated template, but then I have to understand the rules.

Reply:
I added some suggestions to this file. Generally a new line after each phase is how it is done.

Reply:
Done. I hope.